train.py

```diff
@@ -171,13 +250,31 @@ def main():
         # save the model
         model.eval()
         torch.save(model.state_dict(), os.path.join(args.outf, 'net.pth'))
-        #make_sample_images(model)
+        
     # plot loss/epoch for training and validation sets
     training = plt.plot(training_losses, label='training')
     validation = plt.plot(validation_losses, label='validation')
     plt.legend()
     plt.savefig(args.outf + "/loss_plot.png")
+    
+    #write out training and validataion loss values to text files
+    tfileout = open("training_losses.txt","w")
+    vfileout = open("validation_losses.txt","w")
+    for i, elem in enumerate(training_losses):
+        tfileout.write("%f " % training_losses[i] + "\n")
+    for elem in enumerate(validation_losses):
+        tfileout.write("%f " % validation_losses[i] + "\n")
+        vfileout.write("%f " % validation_losses[i] + "\n")
```

Comment:
I'm confused about what these loops are supposed to be doing.

The first loop could just be `for elem in training_losses`, since `training_losses[i]` is `elem`.

The second loop puts the entire output of `enumerate(validation_losses)` in `elem`, then uses `i` instead; but it doesn't define `i`, so it's just reusing the last value of `i` from the previous loop. It's also writing the validation loss into both the validation and training output files.

Reply:
yes this is definitely a mess... I think we had wanted to look at training and validation losses but I didn't finish this so I'll clean that up... 
```suggestion
    for elem in enumerate(training_losses):
        tfileout.write("%f " % elem + "\n")
    for elem in enumerate(validation_losses):
        vfileout.write("%f " % elem + "\n")
```